raiden/connection_manager.py

```diff
@@ -289,3 +320,23 @@ def open_channels(self):
             self.api.get_channel_list(token_address=self.token_address)
             if channel.isopen
         ]
+
+    @property
+    def min_settle_blocks(self):
+        """Returns the minimum necessary waiting time to settle all channels.
+        """
+        channels = self.api.get_channel_list(
+            token_address=self.token_address
+        )
+        timeouts = [0]
```

Comment:
I would have preferred to be more explicit here and say something like:

```python
if not channels:
    return 0
```

But this is nitpicking, so don't feel obliged to change anything.